modules/core/src/lib/base-attribute.js

```diff
@@ -127,6 +127,12 @@ export default class BaseAttribute {
       props.accessor.size = props.size;
     }
     delete props.size;
+    if ('type' in props) {
+      delete props.type;
```

Comment:
Should we put it in `props.accessor` before we delete it?

Reply:
Can we just avoid merging `opts` with `props`? I do not believe the buffer's accessor is used anywhere (the `Attribute` class stores its own accessor) so there's no point setting it.